@types/pal/input.d.ts

```diff
@@ -40,11 +46,134 @@ declare module 'pal/input' {
         public dispatchKeyboardUpEvent? (nativeKeyboardEvent: any);
     }
 
+    export type GamepadCallback = (res: import('cocos/input/types/event').EventGamepad) => void;
```

Comment:
```suggestion
    type GamepadCallback = (res: import('cocos/input/types/event').EventGamepad) => void;
```

we don't export this callback type